content/guides/concepts/cid.md

```diff
@@ -5,18 +5,39 @@ menu:
         parent: concepts
 ---
 
-A *content identifier* is a value that addresses a single piece of content in IPFS. It is mainly a cryptographic hash of the content, but is encoded as a [multihash](https://github.com/multiformats/multihash) and [multicodec](https://github.com/multiformats/multicodec). (Note: older CIDs have a different design — see [version 0](#version-0) below.)
+## Summary
 
-<!-- TODO: explain more of the details of how CID v1 is composed here. -->
+A *content identifier*, or CID, is a label used for addressing content in IPFS. CID's are used as a standard way of pointing to pieces of information.  CID's identify specific pieces of content stored in IPFS.
 
-You can read up on the details in the [CID spec](https://github.com/ipld/cid). You might also want to check out the [CID inspector](http://cid-utils.ipfs.team/#zb2rhiVd5G2DSpnbYtty8NhYHeDvNkPxjSqA7YbDPuhdihj9L) for an interactive breakdown of CIDs.
+CID's are based on that content's cryptographic hash - a different piece of content will have a different hash and will produce different CID's.
```

Comment:
Can make “cryptographic hash” here a link to the concept doc on hashes?

I think it also might be helpful to explain a little more about why this matters, e.g. “Because a CID identifies content by *what it is* rather than by where it is stored, it gives us a way to retrieve the same content from many different peers on the network, rather than just one place — without CIDs, IPFS wouldn’t work at all.”